website/docs/r/network_interface.html.markdown

```diff
@@ -91,11 +92,13 @@ The `ip_configuration` block supports:
 
 * `load_balancer_inbound_nat_rules_ids` - (Optional) List of Load Balancer Inbound Nat Rules IDs involving this NIC
 
+* `primary` - (Optional) Is this the Primary Network Interface? If set to `true` this should be the first `ip_configuration` in the array.
```

Comment:
> The alternative I considered here was just assuming the first would be the primary

Hm, I think making assumptions is _probably_ even worse UX. This is at the very least explicit. So if you think it's ok then go ahead - I just thought as a user I'd be little bit annoyed or confused 🙂